### DISCUSSION TITLE
v4.12 🛠️

### DISCUSSION THREAD

**Discussion by peterpp** [pematon/adminer#7] [Announcements]
### New features

- MySQL: Print comments of stored procedures and functions
- MariaDB: Add support for UUID data type (by vukbgit)
- MS SQL: Allow to set Encrypt and TrustServerCertificate with AdminerLoginSsl plugin (issue #5)
- MS SQL, MongoDB: Connect to localhost with default port if server is not specified
- Compiler: Allow to compile only selected drivers and languages

### Changes

- Change logo link to main page (login)
- Enhance checking of new version
- Update project URL and info
- Rename 'server' driver to 'mysql'
- Compiler: MySQL driver is no longer the required default
- Update Spanish translations (by isaacpolaino)

### Bugfixes

- PostgreSQL: Fix layout of stored function parameters
- MongoDB: Fix parsing WHERE condition from SQL query
- Fix SQL query code direction if RTL language is used
- Fix disappearing dragged row
- Fix highlighting default submit button in indexes form
- Compiler: Fix translations in plugins
- Compiler: Fix compiled SQLite single-driver Adminer

<hr /><em>This discussion was created from the release <a href='https://github.com/pematon/adminer/releases/tag/v4.12'>Released version 4.12 🛠️</a>.</em>

**Comment by mckaygerhard** [comment]
hi @peterpp  i noted after revised a comment at https://github.com/adminerevo/adminerevo/issues/83#issuecomment-2581327296 your version is practically in a new way.. i proposed to change the name and become a new software..

**Reply by peterpp** [reply]
As mentioned in README file, I'm considering to change the project's name. But I'm still not decided :)

**Reply by mckaygerhard** [reply]
**made a conquest** ! people participate and proposed names.. but i experienced that if you open an issue for.. those that comes late lost importance.. so i proposed to put a list in first comment with all the names that are appended to the issue as new comments appears.. i can help on.. of course
